Lib/test/_test_multiprocessing.py

```diff
@@ -6155,3 +6155,9 @@ class SemLock(_multiprocessing.SemLock):
         name = f'test_semlock_subclass-{os.getpid()}'
         s = SemLock(1, 0, 10, name, False)
         _multiprocessing.sem_unlink(name)
+
+    def test_semlock_mixed_context(self):
```

Comment:
Does this test actually need to be Linux-only? (see `SemLockTests` definition above)

Reply:
This is indeed also relevant on macos. Moved the test to be able to test it there.
On windows, there is only spawn, so there is no way to get "mixed" context.